docs/OCP_CI_Tutorials/Containers/Container_Creation_Guide.md

```diff
@@ -11,16 +12,19 @@
     - [Execute in the Container](#execute-in-the-container)
 
 ## Introduction
+
 Prior to discussing the how to use containers in OpenShift CI, it is important to understand the basic structure of a CSPI test in OpenShift CI. 
 
 ```mermaid
     graph TD;
         ocp[OpenShift CI]-->prow[Prow Cluster];
         prow[Prow Cluster]-->test[Test Cluster]
 ```
-<sub><sup>Figure 1: _Cluster Graph_</sup></sub>
+
+`Figure 1: _Cluster Graph`
 
 Figure 1 above shows a simple flow chart to demonstrate that there are really two clusters provisioned when you execute a CSPI job in OpenShift CI.
+
 1. **Prow Cluster**
    - This cluster is the cluster that actually executes the various steps (chains, workflows, refs, _etc._) of a test.
    - This cluster isn't provisioned new each time. There is a new project/namespace in this cluster provisioned and deprovisioned for every execution of OpenShift CI.
```

Comment:
```suggestion
   - The Prow cluster is not newly provisioned each time. There is a new project/namespace in this cluster that provisions and deprovisions for every execution of OpenShift CI.
```

Reply:
Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)